examples/3d/elixir_advection_amr.jl

```diff
@@ -57,6 +55,7 @@ alive_callback = AliveCallback(analysis_interval=analysis_interval)
 analysis_callback = AnalysisCallback(semi, interval=analysis_interval,
                                      extra_analysis_integrals=(entropy,))
 
+
```

Comment:
@ranocha Why the extra blank line here?
```suggestion
```

Reply:
Since that's consistent with the formatting of the remaining elixirs.

Reply:
:+1: